certified-connectors/Azure Communication Services Email/apiProperties.json

```diff
@@ -0,0 +1,23 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Connection String",
+          "description": "Azure Communication Services Connection String.",
+          "tooltip": "Find your connection string in your Azure Communication Services resource in Azure Portal in the \"Keys\" tab.",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
```

Comment:
@lei-msft this color is a default one and it's [not allowed](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector) to use it. Can you please tell me what is the color of your icon and how will it look like? Is the icon Azure-related?